backend/secretary/endpoints/add_new_sport/__init__.py

```diff
@@ -0,0 +1,21 @@
+from flask import request
+from flasgger import SwaggerView
+from verification.jwt import is_secretary
+
+# TODO
+class AddNewSportView(SwaggerView):
+
+	@is_secretary()
```

Comment:
tento dekorator akoze overi nejako, ze ci sa na to pyta sekretarka ? vies mi trochu objasnit, co to robi ?

Reply:
Tu som to nechal ako example, ale chybne - správne je to v `secretary/endpoints/show_sports/__init__.py` bez zátvoriek (opravné v ďalšom commite). 
Tento dekorátor zaobaluje daný route a metódu (v tomto prípade POST) s dvoma funkcionalitami:
1. Overí, či bol v requeste uvedené JWT token a či je validný a neexpirovaný.
2. Ak je token validný, skontroluje jeho typ i.e. userType, ktorý sa ukladá do tokenu pri jeho vytváraní v returne `secretary/endpoints/login/__init__.py`.
Tento dekorátor chceme použiť na každý route a metódu, ktorá má byť zabezpečená, pred neprihláseným používateľom (pre admina to zasa bude `@is_admin`.

Reply:
a kde je definicia toho dekoratora ?

Reply:
`verification/jwt/__init__.py`
Ak používaš Jetbrains produkty e.g. pycharm, dá sa držaním ctrl a kliknutím na dekorátor/funkciu/triedu/... presmerovať na jej zadefinovanie.